.github/workflows/back-tests.yml

```
@@ -0,0 +1,62 @@
+    flutter-version: '3.4.3' # Update to at least 3.4.3 as specified in pubspec.yaml  
+Alternatively, you can set it to a newer stable version if it aligns with your project's requirements.
+
+Update Flutter version and verify project compatibility.
+
+The checkout and Flutter setup steps are correct, but the specified Flutter version (3.0.0) might be outdated. Consider the following:
+
+Update to the latest stable Flutter version or the specific version required by your project.
+Ensure that this version is compatible with your project's dependencies and SDK constraints.
+Example:
+
+- name: Set up Flutter  
```

yamllint

[error] trailing spaces

(trailing-spaces)